scripts/asm-installer/install_asm

```diff
@@ -1598,6 +1627,13 @@ install_secrets() {
     --from-file="${CA_CHAIN}"
 }
 
+init_private_ca() {
+  WORKLOAD_IDENTITY="$PROJECT_ID.svc.id.goog[istio-system/istiod-service-account]"
+  retry 3 run gcloud projects add-iam-policy-binding $PROJECT_ID \
```

Comment:
(discussed offline yesterday) we can now add this binding on the CA itself

Reply:
@pmansour what does that mean? We still using the IAM binding, and the GCP SA to call the CAS, right?

Reply:
We plan to use the 3p identity directly to call the CAS, but what I meant here is that we can grant the IAM role on the CA resource directly (vs. granting it on the parent project resource). This just means a more granular IAM binding which is always good for security :)

Reply:
Thanks for the clarification.